src/api/marketplace/__mocks__/ProductService.js

```diff
@@ -82,7 +82,7 @@ async function getProducts () {
             {
               title: 'reverse-domain-name',
               icon: '',
-              value: 'com.codesys.control'
+              value: 'tech.flecs.mqtt-bridge'
```

Comment:
Almost feel bad for being this picky, but then again...
tech.flecs.service-mesh